Project/Fall2020_CSC403_Project/FrmInventory.cs

```diff
@@ -0,0 +1,167 @@
+﻿using Fall2020_CSC403_Project.code;
+using MyGameLibrary;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Windows.Forms;
+
+namespace Fall2020_CSC403_Project
+{
+    public partial class FrmInventory : Form
```

Comment:
The inventory window currently shows up with a title of "Form1" this can be changed in the designer by selecting the overall window/form and changing the "Text" property.